x-pack/legacy/plugins/ml/public/datavisualizer/datavisualizer_controller.js

```diff
@@ -52,10 +52,11 @@ import { uiModules } from 'ui/modules';
 const module = uiModules.get('apps/ml');
 
 module
-  .controller('MlDataVisualizerViewFields', function ($scope, $timeout, $window, Private, AppState, config) {
+  .controller('MlDataVisualizerViewFields', function ($injector, $scope, $timeout, $window, Private, AppState, config) {
 
     timefilter.enableTimeRangeSelector();
     timefilter.enableAutoRefreshSelector();
+    const mlTimefilterRefreshService = $injector.get('mlTimefilterRefreshService');
```

Comment:
`mlTimefilterRefreshService` could be added to the rest of the injected items, instead of using `$injector`